src/python/WMCore/MicroService/Service/Data.py

```diff
@@ -89,12 +120,25 @@ def get(self, **kwds):
                'microservice_version': WMCore.__version__,  # FIXME: extract it from another place
                'microservice': self.mgr.__class__.__name__}
 
-        if kwds.get('API') == "status":
+        api = kwds.get('API')
+
+        if api == "status":
             detail = True if kwds.pop("detail", True) in (True, "true", "True", "TRUE") else False
             res.update(self.mgr.status(detail, **kwds))
-        elif kwds.get('API') == "info":
+            yield res
+        elif api == "info":
             res.update(self.mgr.info(kwds.pop("request", None), **kwds))
-        return results(res)
+            yield res
+        elif 'pileup' in self.mount:
```

Comment:
Well, api here is MSManager instance under ResApiHub, i.e. `api=<WMCore.MicroService.Service.RestApiHub.RestApiHub object at 0x10d6763d0>` since MSPileup is part of MSManager object..